crypto/merklekeystore/keystore_test.go

```diff
@@ -26,6 +26,7 @@ import (
 	uuid "github.com/satori/go.uuid"
 	"github.com/stretchr/testify/require"
 
+	"github.com/algorand/go-algorand/config"
```

Comment:
Line 61: use `defer` for `signer.keyStore.store.Close()`

Line 117: signer is missing `signer.keyStore.store.Close()`.

Line 156: signer is missing `signer.keyStore.store.Close()`.